tests/models/oneformer/test_modeling_oneformer.py

```diff
@@ -69,6 +70,7 @@ def __init__(
         self.parent = parent
         self.batch_size = batch_size
         self.is_training = is_training
+        self.vocab_size = vocab_size
```

Comment:
Need to accept `vocab_size`: this model uses `CLIPTokenizer`, and the checkpoint on the Hub have tokenizer files. In this case, the creation script will call model tester in order to get a smaller vocab size to use.

Reply:
Add `vocab_size` is also better: we can avoid larger `vocab_size` like `49408` in the testing.